py/client/tests/test_session.py

```diff
@@ -41,6 +41,7 @@ def test_never_timeout(self):
             token2 = session._auth_token
             self.assertNotEqual(token1, token2)
         session.close()
+        sleep(400)
```

Comment:
I assume we skip this because it is going to be very slow and annoy everyone?

Reply:
yes, until we allow the client to set the timeout on the session.